src/frontend/web_application/src/scenes/Timeline2/components/DiscussionItem/index.jsx

```diff
@@ -75,7 +78,7 @@ class DiscussionItem extends PureComponent {
         id={`discussion-${discussionId}`}
         data-nb-messages={total}
         data-date={date}
-        className={`s-discussion-item${unreadCount ? ' is-unread' : ''}`}
+        className={`s-discussion-item${unreadCount ? ' is-unread' : ''} s-discussion-item--${getPiClass(piAggregate)}`}
```

Comment:
it would be easier to maintain with `classnames` library:

```
className={classnames('s-discussion-item', { 'is-unread': unreadCount }, `s-discussion-item--${getPiClass(piAggregate)}`}
```

(and I could be cool to use a modifier like in BEM: `{ 's-discussion-item--is-unread': unreadCount } ...`)